File: src/app/core/models/WorkspaceDetailsModel.ts
import { ChannelDetailsModel } from './ChannelDetailsModel';
import { WorkspaceUserModel } from './WorkspaceUserModel';

export class WorkspaceDetailsModel {
    public Id: string;
    public Name: string;

    public PictureLocation: string;

    public IsOwner: boolean;

    public  Users: WorkspaceUserModel[];

    public Channels: ChannelDetailsModel[] ;
    
}

File: WebAPI/AquaChatAPI.Domain/Models/WorkspaceDetailsModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace AquaChatAPI.Domain.Models
{
    public class WorkspaceDetailsModel
    {
        public WorkspaceDetailsModel(){ }
        public string Id { get; set; }
        public string Name { get; set; }

        public string PictureLocation { get; set; }

        public Boolean IsOwner { get; set; }

        public WorkspaceUserModel[] Users { get; set; }

        public ChannelDetailsModel[] Channels { get; set; }
    }
}
